### DISCUSSION TITLE
DSS loads really slow

### DISCUSSION THREAD

**Discussion by michaelleepow90** [Stellarium/stellarium#1799] [Q&A]
Hello, not sure what's going on but the DSS option loads really slow. I tried zooming into the stars but the images loads really slow.. Is there a way to fix it?

**Comment by gzotti** [comment]
Use a fast Internet connection. If there is no other way, take note of section 10.4.1 of the User Guide.

**Reply by michaelleepow90** [reply]
Cool, thanks. I have 5G fast internet at 75 megabits per second.
